crates/bevy_ecs/src/system/commands/command_queue.rs

```diff
@@ -9,8 +9,11 @@ struct CommandMeta {
     /// SAFETY: The `value` must point to a value of type `T: Command`,
     /// where `T` is some specific type that was used to produce this metadata.
     ///
+    /// `world` is optional to allow this one function pointer to perform double-duty as a drop.
```

Comment:
I've done this to ensure `CommandMeta` is still exactly 8 bytes in size. Since the inner `Drop` implementation for `C` is specific to `C`, this is the only way to capture it without making `CommandMeta` at least 16 bytes large.